app/src/main/java/com/randomappsinc/studentpicker/home/MainActivity.java

```diff
@@ -189,7 +189,9 @@ public void onRenameListConfirmed(int position, String newListName) {
     }
 
     @Override
-    public void onItemDeleteClick(int position) {
+    public void onItemDeleteClick(int position, String listName) {
+        String dialogContent = getString(R.string.confirm_deletion_message) + " \"" + listName + "\"?";
```

Comment:
Once we start using placeholders, the getString() API can actually support it via additional parameters; it's pretty cool.

```suggestion
        String dialogContent = getString(R.string.confirm_deletion_message, " \"" + listName + "\");
```